### PR TITLE
Fixed - lb agent, demuxer and schema_validator unit test cases

### PR DESCRIPTION
**Pull request by pkharat**

### CODE REVIEW

File: gbpservice/neutron/tests/unit/nfp/configurator/test_data/fw_test_data.py [hunk 1]
```diff
@@ -168,6 +168,7 @@ def fake_sa_req_list_fw(self):
                                 ],
                                 "description": '{\
                                     "vm_management_ip": "172.24.4.5",\
+                                    "provider_cidr": "11.0.1.0/24",\
```
Comment (pkharat) [inline gbpservice/neutron/tests/unit/nfp/configurator/test_data/fw_test_data.py:171]: This was needed because test_demuxer.py test case was failing

Comment (njagadish) [inline gbpservice/neutron/tests/unit/nfp/configurator/test_data/fw_test_data.py:171]: Will this break any of the FW test cases?

Comment (pkharat) [inline gbpservice/neutron/tests/unit/nfp/configurator/test_data/fw_test_data.py:171]: This won't. I ran fw test cases. There are some which are failing but not bec of this reason.

**Comment by njagadish** [comment]
Changes are fine.

**Pull request merged**